package.json

```
@@ -98,8 +98,8 @@
     }
   },
   "engines": {
-    "node": "16.15.1",
-    "npm": "8.11.0"
+    "node": "18.17.1",
+    "npm": "9.6.7"
```

```suggestion
    "node": ">=16.15.1 <=18.17.1",
    "npm": ">=8.11.0 <=9.6.7"
```

@pac-guerreiro I think that, for now, we have to specify a range of supported NodeJS versions. I'm thinking about the following situation:

1. This PR is merged and available as a new version to use on E/App
2. Developer runs `npm install` to update dependencies
3. They got an error because `react-native-onyx` is expecting `18.17.1` but E/App currently forces use to use `16.15.1` until we change there as well

After both repos are updated then we can lock this on `18.17.1` only. WDYT?